src/controllers/IBackendController.ts

```diff
@@ -0,0 +1,20 @@
+import { Express } from "express";
+import { ParamsDictionary, RequestHandler } from "express-serve-static-core";
+
+/*
+ * Standard interface for Backend controllers
+ */
+export interface IBackendController {
+  /**
+   * Method used for seting up routing for Controller
+   */
+
+  // tslint:disable-next-line: no-any
```

Comment:
Disabling tslint is usually a sign you're doing something you should not do :)

In this case you can use `unknown` instead of `any`